src/client/handlers/VaultsSecretsWriteFile.ts

```diff
@@ -11,7 +11,7 @@ import * as vaultsUtils from '../../vaults/utils';
 import * as vaultsErrors from '../../vaults/errors';
 import * as vaultOps from '../../vaults/VaultOps';
```

Comment:
This line imports everything from `vaultOps`, but we are only using `writeSecret`. We are needlessly importing things we don't need. While this is consistent with other handlers, should I not do that and only import the singular function I need, or keep it as-is?

Reply:
We usually just import the whole namespace unless we're importing types. Just keep it consistent. AFAIK it doesn't make a difference as far as importing works.